charts/backstage/templates/ingress.yaml

```diff
@@ -20,6 +20,12 @@ spec:
   {{- if .Values.ingress.className }}
   ingressClassName: {{ .Values.ingress.className | quote }}
   {{- end }}
+  {{- if .Values.ingress.tls.enabled }}
+  tls:
+    - hosts:
+      - {{ .Values.ingress.host }}
+      secretName: {{ .Values.ingress.tls.secretName }}
+  {{- end -}}
```

Comment:
```suggestion
  {{- end }}
```

Otherwise the chart fails to render if `ingress.tls.enabled=false`:

It renders:
```
# Source: backstage/templates/ingress.yaml
apiVersion: networking.k8s.io/v1
kind: Ingress
...
spec:rules:
...
```

Reply:
For some reason I thought we had tests in the Actions that would have picked this part up via the install - hence why I lazily didn't run an install locally 😅 . Shall commit the suggestion :+1:

Reply:
Yeah.. no "proper" tests yet.. just deploying the chart without ingress enabled actually :smile: